src/bokeh/models/formatters.py

```diff
@@ -367,21 +367,21 @@ def __init__(self, *args, **kwargs) -> None:
 
     Additionally available variables are:
 
-      * ``ticks``, an array of all axis ticks as positioned by the ticker,
-      * ``index``, the position of ``tick`` within ``ticks``, and
-      * the keys of ``args`` mapping, if any.
+    * ``ticks``, an array of all axis ticks as positioned by the ticker,
+    * ``index``, the position of ``tick`` within ``ticks``, and
+    * the keys of ``args`` mapping, if any.
 
     Finding yourself needing to cache an expensive ``ticks``-dependent
     computation, you can store it on the ``this`` variable.
 
-    Example:
+    Example
 
-        .. code-block:: javascript
+    .. code-block:: javascript
 
-            code = '''
-            this.precision = this.precision || (ticks.length > 5 ? 1 : 2);
-            return Math.floor(tick) + " + " + (tick % 1).toFixed(this.precision);
-            '''
+        code = '''
+        this.precision = this.precision || (ticks.length > 5 ? 1 : 2);
+        return Math.floor(tick) + " + " + (tick % 1).toFixed(this.precision);
+        '''
```

Comment:
Also not sure why this block was dedented. If the google docstring`Example:` header is not working then it would be better to replace "Example" with a full sentence, e.g. "Here is an example of that usage:" or similar. 

FYI before/after screenshots pro-actively attached as PR comments could be extremely helpful in cases like this. Unfortunately I am just not in a place right now with bandwidth to build the docs to check out every change very often.

Reply:
You are right. I don't want to spoil your time, so I will add some images.

- [Current docs](https://docs.bokeh.org/en/latest/docs/reference/models/formatters.html#bokeh.models.CustomJSTickFormatter.code)

![grafik](https://github.com/bokeh/bokeh/assets/68053396/6ac30c30-caaf-4de3-8ca4-200a7ca2329b)

- local build

![grafik](https://github.com/bokeh/bokeh/assets/68053396/41f92445-d82d-4039-b9a2-8135352c6b97)

As you can see, the syntax for `Example:` or `Examples:` is not applied and I can't tell you why. My latest guess is, that this code snipped is inserted after an evaluation process for the key words `Example:` and `Examples`.

This is why I suggest to dedent the block.

Reply:
OK I agree the new change looks better. It would be nice to know why the standard usage is not working as expected, but it's probably not worth the effort. The only small change I might suggest is to make "Example" be bold.